starport/cmd/scaffold_package.go

```diff
@@ -58,11 +59,24 @@ func createPacketHandler(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	signer, err := cmd.Flags().GetString(flagSigner)
```

Comment:
flagGetSigner

Reply:
Why? We always used `flag<FlagName>` for the name of the var for flags

Reply:
I think we should keep the pattern, or I can open a new PR to change all these names to the new pattern

Reply:
that's because we already have it https://github.com/tendermint/starport/pull/1475/files#diff-54341f3927b3c2372b3dbb811dfa708bac37f38d8c6adb8ce35a3a54ac54c386R116-R119